.github/workflows/build-binaries.yml

```diff
@@ -13,11 +18,13 @@ jobs:
       matrix:
         include:
           - os: ubuntu-latest
-            architecture: amd64
-          - os: macOS-latest
-            architecture: amd64
+            architecture: x86_64
+          - os: macos-13
```

Comment:
why it's not latest ? 🤔

Reply:
so we can create a x86 for mac

Reply:
Basically now we will create mac x86, mac arm, linux x86 and linux arm

Reply:
makes sense, thanks